homeassistant/components/huawei_lte/config_flow.py

```diff
@@ -178,12 +191,9 @@ def get_router_title(conn: Connection) -> str:
                     title = info.get("DeviceName")
             return title or DEFAULT_DEVICE_NAME
 
-        username = user_input.get(CONF_USERNAME)
-        password = user_input.get(CONF_PASSWORD)
+        assert self.hass is not None
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6309587)